elasticsearch-details/indices-rule-templates.md

```diff
@@ -0,0 +1,74 @@
+# Templates handling in `indices` rule
```

Comment:
This part is very analytical, like code translated in English. Despite it being precise, it lacks examples entirely and is not goal oriented.
This is in stark contrast with the "index not found" document above, which reads way smoother.

Reply:
@sscarduzio  I've added example in one section. Is it ok and should I continue to add rest of examples in this style?